src/ontology/mondo-edit.obo

```diff
@@ -66808,7 +66808,7 @@ subset: rare
 synonym: "NET of the rectum" EXACT [Orphanet:100081]
 synonym: "neuroendocrine neoplasm of rectum" EXACT [MONDO:patterns/neuroendocrine_neoplasm, NCIT:C5698]
 synonym: "neuroendocrine neoplasm of the rectum" EXACT [NCIT:C5698]
-synonym: "neuroendocrine tumor of rectum" EXACT [DOID:5777, NCIT:C5698]
+synonym: "neuroendocrine tumor of rectum" EXACT [DOID:5777, NCIT:C5698, Orphanet:100081]
```

Comment:
I don't think this synonym should be exact for this term. 
1) NCIT:C5698 does not include it as a synonym (https://www.ebi.ac.uk/ols4/ontologies/ncit/classes?short_form=NCIT_C5698)
2) [orphanet:100081)[https://www.orpha.net/en/disease/detail/100081?name=100081&mode=orpha] this mapping should be reviewed as it might refer to "tumor" and not the more general "neoplasm"
3) this term is very similar to MONDO:0015068-rectal neuroendocrine tumor

Reply:
I agree with your comments. What I think is best we do is

- Go with what NCIT has given its a cancer focused resource
- Orphanet:100081 is to be associated with MONDO:0015068
- All x-refs with Orphanet:100081 on MONDO:0003646 should be removed, as I believe it is referring to MONDO:0015068